### ISSUE TITLE
Fetching scheduled event fails

### ISSUE THREAD

**Issue by pjmurray** [koshilife/calendly-api-ruby-client#16]
When calling `client.scheduled_event` it should return an event as per https://github.com/koshilife/calendly-api-ruby-client/blob/9c862f9257e184b261e97053f3abc9df0771ad63/lib/calendly/client.rb#L129

However I get the following error:

```
NameError: uninitialized constant Calendly::Client::Event
Did you mean?  FSEvent
    app/models/calendly_client.rb:22:in `get_event_from_uri'
    app/controllers/calendly_webhook_controller.rb:36:in `receive'
    lib/jumpstart/lib/jumpstart/account_middleware.rb:30:in `call'
    test/controllers/calendly_webhook_controller_test.rb:64:in `block (3 levels) in <class:CalendlyWebhookControllerTest>'
    test/controllers/calendly_webhook_controller_test.rb:68:in `block (3 levels) in <class:CalendlyWebhookControllerTest>'
```

This looks to be an issue with autloading...
``` 
> Zeitwerk::Loader.eager_load_all
NameError: uninitialized constant Calendly::OrganizationMembership::ModelUtils
```

**Comment by koshilife** [comment]
Hi @pjmurray,

I reproduced it too.
Thanks for your report.
I am going to deal with it as soon as possible.

**Comment by koshilife** [comment]
Hi @pjmurray,

I released fixed version '0.3.0'.
Could you check this issue has been solved?

```
gem 'calendly', '>= 0.3.0'
```

**Comment by pjmurray** [comment]
Hi @koshilife 

Yup! That did the trick :) 

Thank you for the quick turn around
